virt/virt-4-10-release-notes.adoc

```diff
@@ -109,10 +109,10 @@ Deprecated features are included in the current release and supported. However,
 [id="virt-4-10-removed"]
 === Removed features
 
-Removed features are not supported in the current release.
-
 * The VM Import Operator has been removed from {VirtProductName} with this release. It is replaced by the link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.2[Migration Toolkit for Virtualization].
 
+This release removes the template for CentOS Linux 8. CentOS Linux 8 is no longer supported as of December 31, 2021.
+
```

Comment:
This is not part of your PR but I noticed in the preview that the CNV-16642 Dashboard feature release note is incorrectly placed in the Removed features section. This should be moved under the Web console heading, if you don't mind making that change. Thanks.

Reply:
Hi Shikha. This item has already been taken care of by me. It came up in review of the RN. It's now under the Web Console heading

Reply:
yeah I thought that was weird as well. Moving.